spec/classes/corosync_spec.rb

```diff
@@ -637,6 +637,26 @@
     it_configures 'corosync'
   end
 
+  context 'on SLES platforms' do
+    let :facts do
+      {
+        osfamily:        'Suse',
```

Comment:
you also need to mock the `os` hash here. I'm not sure if it's easier or harder to migrate the whole file to rspec-puppet-facts :thinking: